cleanlab/object_detection/rank.py

```diff
@@ -675,6 +712,7 @@ def _compute_badloc_box_scores_for_image(
     lab_bboxes: Optional[np.ndarray] = None,
     similarity_matrix: Optional[np.ndarray] = None,
     min_possible_similarity: Optional[float] = None,
+    has_overlap_label_bboxes: Optional[np.ndarray] = None,
```

Comment:
Does badloc use "has_overlap_label_bboxes"? We should avoid passing this param into functions that don't use it (notice how high_probability_threshold is not passed into this function).